### PR TITLE
Rename mOST to MOST

### LINKED ISSUE #129: Rename mOST to MOST
- This will be the standard to be used in descriptions while mentioning `MOST`; so rename `mOST` to `MOST`
- For the code, we will be using `_most` or `most` as variable names

### PR DESCRIPTION
**Pull request by abhayks1**
PR covers below:

- rename `mOST` to `MOST`
- Update `_most` or `most` as variable names
- Documentation is updated to use `MOST`

Fixes #129

### CODE REVIEW

File: contracts/axiom/Axiom.sol [hunk 1]
```diff
@@ -171,8 +171,8 @@ contract Axiom is AxiomI, ProxyFactory, ConsensusModule {
      * @param _joinLimit Maximum number of validators that can join in a core.
      * @param _gasTargetDelta Gas target delta to open new metablock.
      * @param _coinbaseSplitPerMille Coinbase split per mille.
-     * @param _mOST mOST token address.
-     * @param _stakeMOSTAmount Amount of mOST that will be staked by validators.
+     * @param _most most token address.
```
Comment (pgev) [inline contracts/axiom/Axiom.sol]: MOST token address

File: contracts/reputation/Reputation.sol [hunk 2]
```diff
@@ -71,13 +71,13 @@ contract Reputation is MasterCopyNonUpgradable, ConsensusModule {
 
     /* Storage */
 
-    /** ERC20 mOST for stakes and earnings for validators. */
-    ERC20I public mOST;
+    /** ERC20 most for stakes and earnings for validators. */
```
Comment (pgev) [inline contracts/reputation/Reputation.sol]: ERC20 MOST?

File: contracts/reputation/Reputation.sol [hunk 3]
```diff
@@ -71,13 +71,13 @@ contract Reputation is MasterCopyNonUpgradable, ConsensusModule {
...
+    ERC20I public most;
 
     /** ERC20 wETH for stakes for validators. */
     ERC20I public wETH;
 
-    /** Required stake amount in mOST to stake as a validator. */
+    /** Required stake amount in most to stake as a validator. */
```
Comment (pgev) [inline contracts/reputation/Reputation.sol]: MOST?

File: test/reputation/cashout_earnings.js [hunk 4]
```diff
@@ -147,7 +147,7 @@ contract('Reputation::cashoutEarnings', (accounts) => {
       amount,
       { from: unknownValidator },
     ),
-    'Validator has not staked.');
+      'Validator has not staked.');
```
Comment (pgev) [inline test/reputation/cashout_earnings.js]: What's this spaces are for?

File: test/reputation/cashout_earnings.js [hunk 5]
```diff
@@ -162,7 +162,7 @@ contract('Reputation::cashoutEarnings', (accounts) => {
       amount,
       { from: validator.address },
     ),
-    'Validator is not honest.');
+      'Validator is not honest.');
```
Comment (pgev) [inline test/reputation/cashout_earnings.js]: What's this spaces are for?

File: test/reputation/cashout_earnings.js [hunk 6]
```diff
@@ -179,6 +179,6 @@ contract('Reputation::cashoutEarnings', (accounts) => {
       amount,
       { from: validator.address },
     ),
-    'Validator has withdrawn.');
+      'Validator has withdrawn.');
```
Comment (pgev) [inline test/reputation/cashout_earnings.js]: What's this spaces are for?

File: test/reputation/decrease_reputation.js [hunk 7]
```diff
@@ -140,7 +140,7 @@ contract('Reputation::decreaseReputation', (accounts) => {
       delta,
       { from: constructorArgs.consensus },
     ),
-    'Validator is not active.');
+      'Validator is not active.');
```
Comment (pgev) [inline test/reputation/decrease_reputation.js]: Same.

File: test/reputation/decrease_reputation.js [hunk 8]
```diff
@@ -152,7 +152,7 @@ contract('Reputation::decreaseReputation', (accounts) => {
       delta,
       { from: otherAccount },
     ),
-    'Only the consensus contract can call this function.');
+      'Only the consensus contract can call this function.');
```
Comment (pgev) [inline test/reputation/decrease_reputation.js]: Same.

File: test/reputation/decrease_reputation.js [hunk 9]
```diff
@@ -164,7 +164,7 @@ contract('Reputation::decreaseReputation', (accounts) => {
       delta,
       { from: constructorArgs.consensus },
     ),
-    'Validator is not active.');
+      'Validator is not active.');
```
Comment (pgev) [inline test/reputation/decrease_reputation.js]: Same.

File: test/reputation/decrease_reputation.js [hunk 10]
```diff
@@ -179,7 +179,7 @@ contract('Reputation::decreaseReputation', (accounts) => {
...
```
Comment (pgev) [inline test/reputation/decrease_reputation.js]: Same.

File: test/reputation/decrease_reputation.js [hunk 11]
```diff
@@ -191,6 +191,6 @@ contract('Reputation::decreaseReputation', (accounts) => {
...
```
Comment (pgev) [inline test/reputation/decrease_reputation.js]: Same.

**Comment by abhayks1** [comment]
There are some eslint issues on JS files which I am looking into it.

File: contracts/axiom/Axiom.sol (hunk 1 above)
Comment (abhayks1) [reply contracts/axiom/Axiom.sol]: Done!

File: contracts/reputation/Reputation.sol (hunk 2 above)
Comment (abhayks1) [reply contracts/reputation/Reputation.sol]: Done!

**Comment by abhayks1** [comment]
PR is available for review 📗

**Review by pgev — APPROVED** [review]

**Pull request merged**